### ISSUE TITLE
vis.js - line 3386 - should this be one line?

### ISSUE THREAD

**Issue by fizzer666** [almende/vis#2997]
I had to change this to get timeline examples working.

**Comment by yotamberk** [comment]
Please reference the code explicitly.

**Comment by fizzer666** [comment]
this._dayOfMonthOrdinalParseLenient = new RegExp(
          (this._dayOfMonthOrdinalParse.source || this._ordinalParse.source) +
              '|' + (/\d{1,2}/).source);

Maybe something to do with my line ending settings.  Just caused me problems and wouldn't run until I put it on one line.

**Comment by mbchang** [comment]
I also have an issue with running the timeline examples, and it looks like it is originating from this line as well, although putting the code into one line doesn't seem to help. After installing vis with npm, I try to get this file to work:

```
<!DOCTYPE HTML>
<html>
<head>
  <title>Timeline basic demo</title>
  <script src="js/node_modules/vis/dist/vis.js"></script>
  <link href="js/node_modules/vis/dist/vis.css" rel="stylesheet" type="text/css" />

  <style type="text/css">
    body, html {
      font-family: sans-serif;
    }
  </style>
</head>
<body>
<div id="visualization"></div>

<script type="text/javascript">
  var container = document.getElementById('visualization');
  var data = [
    {id: 1, content: 'item 1', start: '2013-04-20'},
    {id: 2, content: 'item 2', start: '2013-04-14'},
    {id: 3, content: 'item 3', start: '2013-04-18'},
    {id: 4, content: 'item 4', start: '2013-04-16', end: '2013-04-19'},
    {id: 5, content: 'item 5', start: '2013-04-25'},
    {id: 6, content: 'item 6', start: '2013-04-27'}
  ];
  var options = {};
  var timeline = new vis.Timeline(container, data, options);
</script>
</body>
</html>
```
but I get the error:
```
vis.js:3386 Uncaught SyntaxError: Unexpected token this
```

Would anyone have insights on how to fix this?

**Comment by scottwestover** [comment]
> Would anyone have insights on how to fix this?

I had the same issue as well. To fix it, you can try using the vis.js file that is available on cdnjs here: https://cdnjs.cloudflare.com/ajax/libs/vis/4.19.1/vis.js

Another solution that worked for me was to edit vis.js directly (this solution was recommended by @bradh in #2896 :
In the file go to line 3386:
`(this._dayOfMonthOrdinalParse.source || this._ordinalParse.source) +`
and delete the "space" after the || and before this._orginalParse.source so the code looks like this:
`(this._dayOfMonthOrdinalParse.source ||this._ordinalParse.source) +`

**Comment by mbchang** [comment]
Thanks for the feedback. I experimented a little and realized that it also works if you use ```vis.min.js``` instead of ```vis.js```.

**Comment by wimrijnders** [comment]
Duplicate of #2912.
Fixed in `Release 4.4.20.0`.
